gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -25,6 +25,11 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
         assert extension.getSeed() == null
     }
 
+    void testDefaultCategories() {
+        def extension = createExtension()
+        assert extension.getCategories() == AndroidCommandPluginExtension.CATEGORIES_DEFAULT
```

Comment:
I just double-checked, you can just return `null` instead of `CATEGORIES_DEFAULT`, both leads to no `-c` option being passed into the monkey command.